documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1051 @@
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the {rhv-short} storage domain UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+ifdef::ova[]
+[NOTE]
+====
+For OVA, the `StorageMap` can map only a single storage, which all the disks from the OVA are associated with, to a storage class at the destination. For this reason, the storage is referred to in the UI as "Dummy storage for source provider <provider_name>".
+====
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        name: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        name: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the TBD.
+endif::[]
+
+ifdef::ostack[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the {osp} `volume_type` UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+
+ifdef::cnv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the TBD.
+endif::[]
```

Comment:
@bennyz @liranr23 What is TBD here?

Reply:
name is used for the source, not id, and the storage class name is set there